server/config/config.go

```diff
@@ -529,25 +529,44 @@ type ScheduleConfig struct {
 	SchedulerMaxWaitingOperator uint64 `toml:"scheduler-max-waiting-operator,omitempty" json:"scheduler-max-waiting-operator"`
 	// WARN: DisableLearner is deprecated.
 	// DisableLearner is the option to disable using AddLearnerNode instead of AddNode.
-	DisableLearner bool `toml:"disable-raft-learner" json:"disable-raft-learner,string"`
+	DisableLearner bool `toml:"disable-raft-learner" json:"disable-raft-learner,string,omitempty"`
 	// DisableRemoveDownReplica is the option to prevent replica checker from
 	// removing down replicas.
-	DisableRemoveDownReplica bool `toml:"disable-remove-down-replica" json:"disable-remove-down-replica,string"`
+	// WARN: DisableRemoveDownReplica is deprecated.
```

Comment:
Do we need to test them whether echo deprecated ?

Reply:
you mean to check `config show` does not output these flags?

Reply:
And what output when `config set disable* true`

Reply:
ok, as your with ;)